CHANGELOG.rst

```diff
@@ -5,9 +5,17 @@ Changelog
 Version History
 ===============
 
+.. _v1.1.2:
+
+1.1.2 (2017-01-25)
+-------------------
+
+* Reimplements ``get_next_network`` method
+* ``get_next_network`` method now does strict allocation 
```

Comment:
Since `get_next_network()` now _optionally_ does strict allocation, I think this bit in the changelog should be updated to reflect that.